Tests/BasicTest.cs

```diff
@@ -18,6 +18,7 @@ public void GoToMainPage()
             webDriver = new ChromeDriver();
             webDriver.Navigate().GoToUrl("https://www.tritonshoes.ru/");
             mainPage = new MainPage(webDriver);
+            webDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
         }
 
         [TearDown]
```

Comment:
What would your code do if webDriver didn't initialize?

Reply:
Are you sure that it didn't initialize? 
(
public IWebDriver webDriver;
        public MainPage mainPage;
)
It works! Test fall when I don't use it.

Reply:
This is an imaginary situation. Now it works. But if, for some reason wouldn't work. Consider checking webDriver for null in the [TearDown].

> Test fall when I don't use it.
Sure. Did I advice you to remove it)